modules/apps/bean-portlet/bean-portlet-cdi-extension-sample/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0"?>
+
+<project
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd"
+>
+	<modelVersion>4.0.0</modelVersion>
+
+	<!-- Note: The parent POM contains "thin" and "cdi" profiles -->
+
+	<parent>
+		<groupId>com.liferay.faces.archetype</groupId>
+		<artifactId>com.liferay.faces.archetype.parent</artifactId>
+		<version>6.1.0</version>
+	</parent>
+	<groupId>sample.bean.portlet</groupId>
+	<artifactId>sample.bean.portlet</artifactId>
+	<packaging>war</packaging>
+	<name>Sample CDI Bean Project</name>
+	<version>1.0.0</version>
+	<build>
+		<plugins>
+			<plugin>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<configuration>
+					<encoding>UTF-8</encoding>
+					<source>1.8</source>
+					<target>1.8</target>
+				</configuration>
+			</plugin>
+		</plugins>
+	</build>
+	<dependencies>
+		<dependency>
+			<groupId>com.liferay.faces</groupId>
+			<artifactId>com.liferay.faces.bridge.ext</artifactId>
+			<version>6.1.1</version>
+			<scope>${war.dependency.scope}</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.liferay.faces</groupId>
+			<artifactId>com.liferay.faces.bridge.impl</artifactId>
+			<version>4.3.0</version>
+			<scope>${war.dependency.scope}</scope>
+		</dependency>
+		<dependency>
+			<groupId>javax.enterprise</groupId>
+			<artifactId>cdi-api</artifactId>
+			<version>1.2</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>javax.faces</groupId>
+			<artifactId>javax.faces-api</artifactId>
+			<version>2.2</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
+			<groupId>javax.portlet</groupId>
+			<artifactId>portlet-api</artifactId>
+			<version>2.0</version>
+			<scope>provided</scope>
+		</dependency>
```

Comment:
@Kvale1733, please use portler-api 3.0.1 because master is using 3.0.1. In addition,  other dependencies version are a little old. For example, [com.liferay.faces.bridge.ext](https://mvnrepository.com/artifact/com.liferay.faces/com.liferay.faces.bridge.ext). Please try to use the new version to build the war.